lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -468,15 +469,15 @@ spec = do
             \Address."
         negativeTest proxy ".%14'"
             ("Unable to decode Address: encoding is neither Bech32 nor Base58.")
-        negativeTest proxy "ca1qvqsyqcyq5rqwzqfpg9scrgk66qs0"
+        negativeTest proxy "ca1qv8qurswpc8qurswpc8qurs7xnyen"
```

Comment:
Had to regenerate this case since it was using a first byte that we now correctly recognize :)